sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -69,7 +69,7 @@ public class EventProcessorClient
 
                 if (_partitionInitializingAsync != default)
                 {
-                    throw new NotSupportedException("TODO: no more than one event.");
+                    throw new NotSupportedException("TODO: A handler has already been assigned to this event.");
```

Comment:
The resources file is all linked in and shared.   We should make sure that we tune these messages for release - whether they're resource-driven or hardcoded.